graphics/vips/Portfile

```diff
@@ -5,8 +5,8 @@ PortGroup           github 1.0
 PortGroup           gobject_introspection 1.0
 PortGroup           meson 1.0
 
-github.setup        libvips libvips 8.15.1 v
-revision            4
+github.setup        libvips libvips 8.15.2 v
+revision            1
```

Comment:
I may miss something, but if there was no 8.15.2 earlier, revision should be reset to 0.